src/hacking/building-for-openharmony.md

```diff
@@ -196,24 +36,30 @@ Otherwise, the environment variables will not be loaded.
 The following command can then be used to compile the servoshell application for a 64-bit ARM device or emulator:
 
 ```commandline
-./mach build --ohos --release [--flavor=harmonyos]
+./mach build --ohos --release --flavor=harmonyos
```

Comment:
The flavor is optional (and defaults to openharmony), so i would keep the brackets